BinaryEncoding.md

```diff
@@ -67,18 +67,30 @@ Yes:
 
 * A module contains:
   * a header followed by
-  * a table (sorted by offset) containing, for each section, its type and offset (within the module), followed by
+  * a table (sorted by offset) containing, for each section, its type and
+    offset (within the module), followed by
   * a sequence of sections.
 * A section contains:
   * a header followed by
   * the section contents (specific to the section type)
+* A definitions section contains:
+  * the generic section header
+  * a table containing, for each opcode-space, a standardized string literal
+    type name (where index defines its type), offset (within the section),
+    sorted by offset, followed by
```

Comment:
Could you make these sub-bullets?

What do you mean by "index defines its type"?